tasks/main.yml

```diff
@@ -1 +1,31 @@
 # role tasks
+
+- name: Install necessary packages
+  package:
+    name: parted
+    state: present
+  become: "{{ disk_wipe_become }}"
+  become_user: "{{ disk_wipe_become_user }}"
+
+- name: Clear the disk signatures
+  become: "{{ disk_wipe_become }}"
+  become_user: "{{ disk_wipe_become_user }}"
+  command: "wipefs --all --force {{ item }}"
+  register: signatures_before
+  changed_when: '"offset" in signatures_before.stdout'
+  with_items: "{{ disk_wipe_disks }}"
+
+- name: Re-read partion tables
```

Comment:
This should probably run at the end, given that we are performing more disk write activities _after_ this.  Even so, is this necessary in the context of just `wipefs`?  From the `wipefs` man page:

> wipefs calls the BLKRRPART ioctl when it has erased a partition-table signature to inform the kernel about the change.

Reply:
re-ordered role to run shred, wipefs no longer a need for parted to re-read partition tables.